### PR TITLE
Test on PHP 8.1

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,7 +44,14 @@ script:
   - diff -B phpcs.mo4.xml <(xmllint --format phpcs.mo4.xml)
   - diff -B phpcs.xml.dist <(xmllint --format phpcs.xml.dist)
   - vendor/bin/phpcs -s --standard=MO4 integrationtests/testfile.php # Check, if all the sniffs work with the current dependencies
-  - vendor/bin/phpunit # Run tests
+  # Another PHPUnit workaround, this time, because of PHPCS itself
+  - |
+    if [ "${TRAVIS_PHP_VERSION}" != "8.1.0" ]; then
+      export NEXT_PHPUNIT_WORKAROUND=""
```
Comment (xalopp) [inline .travis.yml:50]: I suggest to simplify the new `if-then-elif ` to a simple `if-then-else`
```
if [ "${TRAVIS_PHP_VERSION}" = "8.1.0" ]; then
    export NEXT_PHPUNIT_WORKAROUND="--no-configuration --bootstrap=tests/bootstrap.php --dont-report-useless-tests vendor/squizlabs/php_codesniffer/tests/Standards/AllSniffs.php"
else
    export NEXT_PHPUNIT_WORKAROUND=""
fi
```

**Review by xalopp — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (mmoll) [reply .travis.yml]: Should be addressed now.

**Comment by mmoll** [comment]
Our mayflower org is currently out of credits on Travis. Either we switch to GH actions or we need to investigate, which repo exceeded the current limits.

**Comment by mmoll** [comment]
Damn, we need to to switch to GH actions anyway... https://docs.travis-ci.com/user/billing-overview/#free-trial-plan

**Review by xalopp — APPROVED** [review]
Awesome! Merci @mmoll

**Review by xalopp — APPROVED** [review]

**Pull request merged**